<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Failed
- hook id: remove-tabs
- exit code: 1
- files were modified by this hook

Substituting tabs in: policy/argocd_app.rego by 4 whitespaces

Tabs have been successfully removed. Now aborting the commit.
You can check the changes made. Then simply "git add --update ." and re-commit

Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Check for added large files..............................................Passed
Check for case conflicts.................................................Passed
Check JSON...........................................(no files to check)Skipped
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 2

.github/workflows/conftest.yaml
  1:1       warning  missing document start "---"  (document-start)
  3:1       warning  truthy value should be one of [false, true]  (truthy)

ml-prague-2021/kustomization.yaml
  6:3       warning  comment not indented like content  (comments-indentation)

.aicoe-ci.yaml
  1:1       warning  missing document start "---"  (document-start)
```
</details>

--- a/.aicoe-ci.yaml
+++ b/.aicoe-ci.yaml
@@ -1,0 +1,3 @@
+---
+check:
+  - thoth-precommit

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -1,0 +1,33 @@
+---
+exclude: .github/workflows
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.1
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-json
+      - id: check-symlinks
+      - id: detect-private-key
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.23.0
+    hooks:
+      - id: yamllint
+        files: \.(yaml|yml)$
+        types: [file, yaml]
+        entry: yamllint --strict

--- a/ml-prague-2021/kustomization.yaml
+++ b/ml-prague-2021/kustomization.yaml
@@ -1,7 +1,6 @@
 ---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
-resources:
-  # Append your application to this list
-  # - template-application.yaml
+# Append your application to this list
+# resources:
+#   - template-application.yaml

--- a/policy/argocd_app.rego
+++ b/policy/argocd_app.rego
@@ -1,15 +1,15 @@
 package main
 
 deny[msg] {
-	input.kind == "Application"
-	input.spec.project != "workshops"
+    input.kind == "Application"
+    input.spec.project != "workshops"
 
-	msg := "Application must use the 'workshops' project"
+    msg := "Application must use the 'workshops' project"
 }
 
 deny[msg] {
-	input.kind == "Application"
-	not startswith(input.spec.destination.namespace, "ws-")
+    input.kind == "Application"
+    not startswith(input.spec.destination.namespace, "ws-")
 
-	msg := "Application resources must target the 'ws-*' namespaces only"
+    msg := "Application resources must target the 'ws-*' namespaces only"
 }

